bugmon_tc/monitor/tasks.py

```diff
@@ -103,41 +65,136 @@ def task(self):
                 "maxRunTime": MAX_RUNTIME,
             },
             "priority": "high",
-            "workerType": self.WORKER_TYPE,
+            "workerType": self.worker_type,
             "retries": 5,
             "routes": ["notify.email.jkratzer@mozilla.com.on-failed"],
             "schedulerId": "-",
             "scopes": self.scopes,
             "tags": {},
         }
 
+    @property
+    def scopes(self):
+        """Scopes applied to the task"""
+        return [
+            "docker-worker:capability:device:hostSharedMemory",
+            "docker-worker:capability:device:loopbackAudio",
+            "docker-worker:capability:privileged",
+            "queue:scheduler-id:-",
+        ]
+
+    @property
+    @abc.abstractmethod
+    def worker_type(self):
+        """The worker type to use for this task"""
+        pass
 
-class ReporterTask(ProcessorTask):
-    """
-    Helper class for generating reporter tasks
-    """
 
-    TASK_NAME = "Reporter Task"
-    WORKER_TYPE = "bugmon-monitor"
+class ProcessorTask(BaseTask):
```

Comment:
Does processor task need `privileged` and `disableSeccomp`  capabilities added to the `task` property?

Reply:
Yes.